integration_test/tests/wallet.rs

```diff
@@ -240,6 +261,25 @@ fn wallet__get_transaction__modelled() {
     model.unwrap();
 }
 
+#[cfg(not(feature = "v17"))]
+#[test]
+fn wallet__list_received_by_label__modelled() {
+    let node = Node::with_wallet(Wallet::Default, &[]);
+    node.fund_wallet();
+    let label = "test-label";
+
+    // Send some coins to the label
+    let amount = Amount::from_sat(10_000);
+    let address = node.client.new_address_with_label(label).unwrap().assume_checked();
+    let _ = node.client.send_to_address(&address, amount).unwrap();
+    node.mine_a_block();
+
+    let json: ListReceivedByLabel = node.client.list_received_by_label().expect("listreceivedbylabel");
+    let model: Result<mtype::ListReceivedByLabel, ListReceivedByLabelError> = json.into_model();
+    let model = model.unwrap();
+    assert!(model.0.iter().any(|item| item.label == label));
+}
```

Comment:
Oh, you used my suggestion here but didn't follow along in `wallet__get_received_by_label__modelled`, my bad I thought you'd notice that.

Reply:
I got thoroughly confused just then, I swear I changed this, but then realised that it was wallet__list_received_by_label__modelled. They are so similar I think my brain just merged them into one. Definitely my bad I missed it.

Reply:
No stress, I got confused too actually when checking it.